src/main/io/serial_cli.c

```diff
@@ -500,6 +500,10 @@ static const char * const lookupTableLowpassType[] = {
     "NORMAL", "HIGH"
 };
 
+static const char * const lookupTableLedAux[] = {
+    "OFF", "1","2","3","4"
```

Comment:
I am not sure this is needed as a lookuptable, is it not fine as an uint instead ? 
Also the limit should be MAX_AUX_CHANNELS and not 4.